doc/developers/contributing.rst

```diff
@@ -229,7 +229,7 @@ how to set up your git repository:
 
        $ pip install --no-build-isolation --editable .
 
-   for more details about advanced installation, see the
+   If you receive errors in building scikit-learn, follow steps 2-5 in the
```

Comment:
I'd rather not mention the section number becasue these may change and they'd become out of sync

```suggestion
   If you receive errors in building scikit-learn, see the
```

Reply:
Thanks for the suggestion @NicolasHug !  LGTM!